python/helpers/pydev/pydevd_file_utils.py

```diff
@@ -73,10 +73,29 @@
 
 normcase = os_normcase # May be rebound on set_ide_os
 
+def convert_to_long_pathname(filename):
```

Comment:
Hi,

I'm not sure what you mean here. Do you just want me to replace 'def' with assignment? Could you provide an example?